lib/hanami/model.rb

```diff
@@ -73,6 +73,7 @@ def self.container
 
     # @since 0.1.0
     def self.load!(&blk) # rubocop:disable Metrics/AbcSize
+      configuration.gateway.use_logger(configuration.logger)       unless configuration.logger.nil?
```

Comment:
@mereghost @pascalbetz By avoiding the call to `use_logger`, we avoid the overhead of thousands of useless method calls against a _null object_.

Because once `gateway` receives a logger, it uses it. It wouldn't log, because it's a _null object_, but still it invokes methods on it. Which is a useless overhead.

WDYT?